src/components/FIRAPoints.tsx

```diff
@@ -0,0 +1,67 @@
+import { Table, TableHead, TableRow, TableCell, TableBody, Stack } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { Results } from 'crewtimer-common';
+import React from 'react';
+import { firaPointsCalc } from '../calculators/FIRAPointsCalc';
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  '&:nth-of-type(odd)': {
+    backgroundColor: theme.palette.action.hover,
+  },
+  // hide last border
+  '&:last-child th': {
+    border: 0,
+  },
+}));
+
+const HeaderTableCell = styled(TableCell)(() => ({
+  fontWeight: 'bold',
+}));
+
+export const FIRAPointsTraditional: React.FC<{ results: Results }> = ({ results }) => {
+  const points = firaPointsCalc(results);
+
+  const maxDataRows = Math.max(points.men.length, points.women.length, points.overall.length);
+  const tableBody: JSX.Element[] = [];
+  for (let row = 0; row < maxDataRows; row++) {
+    tableBody.push(
+      <StyledTableRow key={row}>
+        <TableCell></TableCell>
+        <TableCell align='left'>{row + 1}</TableCell>
```

Comment:
The row index appears to be doubling as the 'Place'.  What happens when there is a tie so there may be one first, two second places, and no third but fourth, fifth, etc  There is a helper function in CrewTimerUtils for assigning place based on a list of numbers (points).  Does your points awarding consider ties as well?  If not, you may need to use this on the raw milliseconds timestamps to determine ties in individual races.

Your code might be something like the following to assign place based on awarded points:
```
// Feed genPlaces an array of awarded points.  Result will be corresponding array of 'place' for each index.
const places = genPlaces(
    points.men.map((entry) => entry.points),
    'desc',
  );
  // inject the place back into data structure
  places.forEach((place, i) => (points.men[i].place = place));
```

You would then use points.men[row].points when showing the table.

Reply:
All great feedback. I must admit, I did not account for ties and will update some of the unit testing to force that situation (I was using basically live data and started to change it to tickle other corner cases - but ties was not one of them). I'll take a look at things and figure out how to best handle things. I had "borrowed" code from the Barnes system, so that's how I ended up with overloading the "row" as place.

Reply:
So regarding ties, should I break apart the tables so places can be shown individually (because ties will most likely only occur in only a single category - not across all three) or do you think it's okay to embed the places into the same table (e.g., the headers would be: "Place   Overall    Place  Women    Place  Men")